modules/ossm-select-inplace-strategy.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: Concept
+[id="select-inplace-strategy"]
+= Select InPlace strategy
+:context: ossm-select-inplace-strategy
+
+To select the `InPlace` strategy, set the `spec.updateStrategy.type` value in the {istio} resource to `InPlace`.
+
+[source,yaml, subs="attributes,verbatim"]
+----
+kind: Istio
+spec:
+  updateStrategy: 
+    type: InPlace
+----
+
+You can set this value when you first create the resource or you can edit the resource later. If you choose to edit the resource after it is created, make the change prior to updating the {istio} control plane.
+
+When the {istio} is configured to use the `InPlace` strategy, the `IstioRevision` resource that the Operator creates always has the same name as the {istio} resource.
```

Comment:
This should be the first sentence under "Attach workloads to the control plane" instead of being here.

Reply:
done